test/check-application

```diff
@@ -243,7 +243,53 @@ class TestFiles(testlib.MachineCase):
         # Changing hostname updates breadcrumb
         self.restore_file("/etc/hostname")
         m.execute("hostnamectl set-hostname files")
-        b.wait_text(".breadcrumb-button:nth-of-type(1)", "files")
+        b.wait_text(".breadcrumb-button:nth-of-type(2)", "files")
+
+        # Navigation via editing the path
+        path_input = "#new-path-input"
+        edit_button = ".breadcrumb-edit-button"
+        apply_button = ".breadcrumb-edit-apply-button"
+        cancel_button = ".breadcrumb-edit-cancel-button"
+
+        # Cancel
+
+        # Via escape
+        b.click(edit_button)
+        b.wait_val(path_input, "/home/admin")
```

Comment:
Please change the value here, and then make sure that the currently shown path does *not* actually change after cancelling.